apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/EventPropertiesActivity.java

```diff
@@ -87,52 +88,76 @@ public boolean onCreateOptionsMenu(Menu menu) {
 
     @Override
     public boolean onOptionsItemSelected(MenuItem item) {
-        switch (item.getItemId()) {
-            case R.id.action_add:
-                final TextView keyView = mAddPropertyLayout.findViewById(R.id.key);
-                final TextView valueView = mAddPropertyLayout.findViewById(R.id.value);
-                keyView.setText("");
-                valueView.setText("");
-                mAddPropertyLayout.setVisibility(View.VISIBLE);
-                mAddPropertyLayout.findViewById(R.id.add_button).setOnClickListener(new View.OnClickListener() {
-
-                    @Override
-                    public void onClick(View v) {
-                        CharSequence key = keyView.getText();
-                        CharSequence value = valueView.getText();
-                        if (!TextUtils.isEmpty(key) && !TextUtils.isEmpty(value)) {
-                            getSelectedTarget().setEventProperty(key.toString(), value.toString());
-                            updatePropertyList();
+        AnalyticsTransmissionTarget target = getSelectedTarget();
```

Comment:
Add todo comment to remove reflection after release.

Maybe select the method as it was before for jCenter? Does the app run without crashing on jCenter?
An alternative to reflection is to have the 2 files in 2 build variants (2 versions of the file), we used that in the past when reflection was becoming too complex.

Reply:
Using reflection was a bit of a pain here bit since it's already being used I think it's OK to continue the pattern.  As long as we keep on top of removing it we should be fine.  Is this already documented in our release process?

Reply:
It is documented yes, but also mentions to check for files in multiple build variants as this is also a pattern we used in the past.